### PR TITLE
update readme.md

### PR DESCRIPTION
**Pull request by i10416**

### CODE REVIEW

**Comment by SethTisue** [comment]
It's so easy to forget to update this, perhaps it would be better to just put `<version>` and expect the user to fill it in.

File: readme.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Next, install this plugin in `project/plugins.sbt`
 
 ```scala
 // sbt 1 only, see FAQ for 0.13 support
-addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.9")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.10")
```
Comment (i10416) [inline readme.md:86]: > 
```suggestion
addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "<version>")
```

**Comment by i10416** [comment]
> it would be better to just put <version>

Yes.

I just replaced it with `<version>`

**Review by SethTisue — APPROVED** [review]

**Comment by olafurpg** [comment]
I've generally preferred having a version that can be copy-pasted and it works out of the box instead of a placeholder like `<version>` that requires users to figure out the actual version. The former feels more friendly to beginners who don't know the tricks to figure out the release version while it's normally not a big issue if you're using 1.5.9 instead of 1.5.10.

This is not something I feel strongly about, just wanted to share why I think it's generally fine to list a semi-outdated version in the readme. The maven badge above the instructions includes the latest version anyways

CleanShot 2022-01-30 at 08 56 25@2x

**Comment by eed3si9n** [comment]
SGTM. I'll be happy to hit merge if either wants to send a PR for it :)

**Comment by olafurpg** [comment]
Having the placeholder is fine, ignore my rambling 😅

**Pull request merged**